### DISCUSSION TITLE
live stream of recoded video file?

### DISCUSSION THREAD

**Discussion by hardikthakkar2208** [HaishinKit/HaishinKit.swift#1358] [Q&A] [answered]
Can this library give support to stream recoded video files on youtube?

**Comment by shogo4405** [comment, marked as answer]
I haven't tried it yet, but there is a possibility that using [AVAssetReader](https://developer.apple.com/documentation/avfoundation/avassetreader) to extract raw data and then utilizing `stream.append(_ sampleBuffer: CMSampleBuffer)` could be a viable option.

**Comment by hardikthakkar2208** [comment]
Thank you for your replay, I will try using this. If possible please add this option in future releases. Thank you
